schedule/utils.py

```diff
@@ -183,11 +187,11 @@ def decorator(request, *args, **kwargs):
         return HttpResponseNotFound('<h1>Page not found</h1>')
     return decorator
 
+
 def check_calendar_permissions(function):
     @wraps(function)
     def decorator(request, *args, **kwargs):
         if CALENDAR_VIEW_PERM:
-            from schedule.models import Event, Calendar, Occurrence
```

Comment:
Hmm. Those models aren't used in the code below. So I'm curious how this could happen.

Do you have an example code snippet that demonstrates the recursive import? If so I can investigate it more.